### ISSUE TITLE
Improve SVG upload

### ISSUE THREAD

**Issue by shimsa** [elementor/elementor#8528]
## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest stable version of Elementor.

## Description

Recently I use Safe SVG plugin, which now Elementor replacing.
But there are some differences, in which is Safe SVG better.

## Steps to reproduce

When uploading SVG file in the media library
(See images bottom. First is English pictures in order Safe SVG first, Elementor second):
1. Elementor does not show resolution of the SVG picture.
2. Elementor does not round off size of the SVG file, which not showing well especially in Czech language.

![safe-svg-english](https://user-images.githubusercontent.com/7617350/61007401-04df0c00-a36d-11e9-8e1e-ab437d6ab2d8.png)
![elementor-english](https://user-images.githubusercontent.com/7617350/61007398-04df0c00-a36d-11e9-8b34-dd5a7c693278.png)
![safe-svg-czech](https://user-images.githubusercontent.com/7617350/61007400-04df0c00-a36d-11e9-8ec6-099083cd453d.png)
![elementor-czech](https://user-images.githubusercontent.com/7617350/61007397-04467580-a36d-11e9-894d-664128effadc.png)

## Isolating the problem

- [x] This bug happens with only Elementor plugin active (and Elementor Pro).
- [x] This bug happens with a default WordPress theme active.
- [x] I can reproduce this bug consistently using the steps above.

**Comment by arielk** [comment]
As far as I know the WP Media Library does not show the dimensions of SVG files, can you please send us the SVG file?
<img width="1228" alt="Screen Shot 2019-07-19 at 12 36 16" src="https://user-images.githubusercontent.com/1778512/61525913-09b55700-aa22-11e9-9e80-6a9917b3fd43.png">

**Comment by shimsa** [comment]
That's right. But Safe SVG plugin adds this useful function. Why could Elementor not support this?

I am also attaching link to SVG file, which have size 1.008 B and with Elementor its size showing with `&nbsp;` in (WordPress) Czech language. 1 KB would be better.

https://fontawesome.com/icons/accusoft?style=brands

**Comment by shimsa** [comment]
Is there any progress? After creating this issue, Elementor disabled uploading SVG files straight in Media Library. Uploading only through Elementor Editor is now allowed. But issues is still persist.

You need SVG file with size like 1 041 bytes (count with Elementor sanitization wich may downgrade the size of the file) and when you upload it `&nbsp;` in size info is showing. Save SVG plugin solving this, why not Elementor?

And showing resolution of the SVG image is something what is usefull.

Please look at this and fix it.
